rebar.config

```diff
@@ -10,8 +10,8 @@
                            {ref, "eb68fe3"}}},
         {aebytecode, {git, "https://github.com/aeternity/aebytecode.git",
                            {tag, "v3.1.1"}}},
-        {aesophia, {git, "https://github.com/aeternity/aesophia.git",
-                    {tag, "v7.0.1"}}},
+        {aesophia, {git, "https://github.com/davidyuk/aesophia.git",
+                    {ref, "f4e0b56cfbf8dcd300339dd231717e2bb2e120c3"}}},
```

Comment:
Refer to this commit instead https://github.com/aeternity/aesophia/commit/4ca90feea0f70b6a06e811ab5b7892518e97ff65